cmd/optimize/events.go

```diff
@@ -460,6 +482,32 @@ func listRecommendations(flags *recommendationsCmdFlags) func(*cobra.Command, []
 			_, next_ok = data_set.Links["next"]
 		}
 
+		// extract recommendation optimizer ID + num to query and append blocker data from optimization_started events
+		for i := range recommendationRows {
+			blockerRows, err := getOptimizationBlockerData(recommendationRows[i], flags)
+			if err != nil {
+				return fmt.Errorf("Could not get optimization_started data association with recommendation: %w", err)
```

Comment:
This should log a warning that includes the optimizer ID and timestamp for the given row to account for cases where the recommendation was emitted within the range of `since` but the optimization_started event was emitted before the `since`